content/blog/2023-03-15-the-ethereum-classic-mining-difficulty-adjustment-explained/index.md

```diff
@@ -0,0 +1,135 @@
+
+For new entrants, the way of knowing if they are joining the right network is just by checking the blockchain with the most accumulated work done.
+
+## However, Changes in Hashrate Messes Things Up
+
+But, what would happen if the dedicated computing power increases or decreases significantly?
+
+As Bitcoin has a target block time of 10 minutes, Ethereum Classic has a target block time of 13 seconds. This means that a block, together with its new issuance of coins as a reward for miners, is built on average every 13 seconds stabilizing the creation of the money and the amount of data accumulated by the network. 
+
+Block times would become either very fast or very slow with increases or decreases in computing power dedicated to the system. This may happen as computing becomes more efficient with innovation as time passes, when miners just buy more machines and point them to the network, or when they decide to turn off their machines because profitability is low.
+
+If blocks become very fast, then the issuance of the currency would increase, raising inflation, and bloating the network with too many blocks. If blocks become very slow, the issuance would be very deflationary, and the use of the network would become impractical as transactions would take too long to execute.
+
+## The Solution Is the Difficulty Adjustment
+
+![It is difficult to get the cryptographic stamp on target.](./etc-difficulty-banner.png)
+
+The solution to keep the blocks at a steady and stable rate is by adjusting the mining difficulty.
+
+As said before, the hashrate of ETC may grow when machines become more efficient as time passes by, or when miners just buy more machines increasing the computing power dedicated to the network. 
+
+When this happens, blocks may become much faster than 13 seconds. To slow them down again, the network adjusts the difficulty higher so it may take about 13 seconds again, on average, to build blocks.
+
+When hashrate shrinks for any reason, then blocks become too slow. To speed them up, the network adjusts the difficulty lower so it may fine tune the block times back to around 13 seconds, on average. 
+
+## How Proof of Work Works
+
+![Hitting the PoW target.](./1.png)
+
+The best way to explain the difficulty adjustment is by explaining how proof of work works. It all start with miners receiving and preparing the block data. When they have it together, then they hash the data to create a cryptographic stamp for that information. When they create the stamp, they check if the stamp hit a target imposed by the network. If the target is not hit, which happens the great majority of times, then miners have to go back and re-hash the information, producing a new stamp, that they have to check again. This may happen quadrillions of times per second and this is why there is so much work involved, with so much computing power and electricity usage in PoW mining.
```

Comment:
Line 89: Correct typo
```suggestion
The best way to explain the difficulty adjustment is by explaining how proof of work works. It all starts with miners receiving and preparing the block data. When they have it together, then they hash the data to create a cryptographic stamp for that information. When they create the stamp, they check if the stamp hit a target imposed by the network. If the target is not hit, which happens the great majority of times, then miners have to go back and re-hash the information, producing a new stamp, that they have to check again. This may happen quadrillions of times per second and this is why there is so much work involved, with so much computing power and electricity usage in PoW mining.
```